.gitignore

```diff
@@ -73,3 +73,6 @@ bench-restore
 restore-time.txt
 restore.log
 test-output
+
+## Benchmarks compressed data
```

Comment:
```suggestion
## Benchmarks uncompressed data
```

The `.tgz` file is part of the git repository, right — it's just the uncompressed folder that we exclude here?

Reply:
yeah, I kind of gave it a different meaning, data that are compressed in the benchmark, but ok, your interpretation is more natural